Xamarin.Forms.Platform.Android/AppCompat/NavigationPageRenderer.cs

```diff
@@ -311,7 +311,7 @@ protected override void OnAttachedToWindow()
 			RegisterToolbar();
 
 			// If there is already stuff on the stack we need to push it
-			PushCurrentPages();
+			PushCurrentPagesAsync();
```

Comment:
Here, should wait for the Task to complete, and depending on the returned value (bool), update or not the Toolbar?